src/arallocf.f

```diff
@@ -562,5 +566,16 @@ SUBROUTINE ARALLOCF
         ALLOCATE( CRTAG(MXRST), STAT=iost )
         IF ( iost .ne. 0 ) CALL BORT( BRTSTR // 'CRTAG' )
 
+C       MODA_H4WLC arrays.
+
+        ALLOCATE( LUH4WLC(MXH4WLC), STAT=iost )
+        IF ( iost .ne. 0 ) CALL BORT( BRTSTR // 'LUH4WLC' )
+
+        ALLOCATE( STH4WLC(MXH4WLC), STAT=iost )
+        IF ( iost .ne. 0 ) CALL BORT( BRTSTR // 'STH4WLC' )
+
+        ALLOCATE( CHH4WLC(MXH4WLC), STAT=iost )
```

Comment:
This is not just testing, right?

Did you also find and fix some bugs? Or refactor?

Reply:
I decided to include MXH4WLC as one of the settings that can be configured by the user via isetprm, rather than fixed at some arbitrary parameter value as before.  This did make it easier to test the remaining statements in hold4wlc, but I also think it's a useful overall upgrade to the code, and which is why I went ahead with it.

So this wasn't to fix any sort of bug.  Instead, it was just an upgrade to make the library a bit more flexible/configurable for users.

Reply:
It would be good in such a case to add a new issue which describes  the refactor/improvement.